sos/collector/clusters/satellite.py

```diff
@@ -0,0 +1,39 @@
+# Copyright Red Hat 2020, Jake Hunsaker <jhunsake@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from pipes import quote
+from sos.collector.clusters import Cluster
+
+
+class satellite(Cluster):
+    """Red Hat Satellite 6"""
+
+    cluster_name = 'Red Hat Satellite 6'
+    packages = ('satellite', 'satellite-installer')
+
+    def _psql_cmd(self, query):
+        _cmd = "su postgres -c %s"
+        _dbcmd = "psql foreman -c %s"
+        return _cmd % quote(_dbcmd % quote(query))
+
+    def get_nodes(self):
+        cmd = self._psql_cmd('select name from smart_proxies')
```

Comment:
If you have here `copy (select name from smart_proxies) to stdout`, no need to bother by header and footer in the response (one less `idx` shuffling in beow code):

    # su - postgres -c "psql foreman -c 'select name from smart_proxies'"
                         name                      
    -----------------------------------------------
     pmoravec-sat66-on-rhev.gsslab.brq2.redhat.com
     pmoravec-caps66-rhev.gsslab.brq2.redhat.com
    (2 rows)
    
    # su - postgres -c "psql foreman -c 'copy (select name from smart_proxies) to stdout'"
    pmoravec-sat66-on-rhev.gsslab.brq2.redhat.com
    pmoravec-caps66-rhev.gsslab.brq2.redhat.com
    #

But as the current code works well..